src/index.js

```diff
@@ -0,0 +1,292 @@
+// importing dependencies - DO NOT REMOVE
+const mysql = require("mysql2");
+const inquirer = require("inquirer");
+const { userOptions, departmentInfo, deleteRecord } = require("./questions.js");
+const Db = require("./lib/Db");
+const table = require("table");
```

Comment:
Try the console.table npm package instead: https://www.npmjs.com/package/console.table